recipe/build.sh

```diff
@@ -24,8 +24,8 @@ if [ $(uname) == Linux ]; then
     export CPPFLAGS="$CPPFLAGS -DFC_WEIGHT_EXTRABLACK=215 -DFC_WEIGHT_ULTRABLACK=FC_WEIGHT_EXTRABLACK"
     export CPPFLAGS="$CPPFLAGS -DGLX_GLXEXT_PROTOTYPES"
 else
-    compiler_mkspec=mkspecs/common/clang.conf
-    flag_mkspec=mkspecs/macx-g++42/qmake.conf
+    compiler_mkspec=mkspecs/unsupported/macx-clang-libc++/clang.conf
```

Comment:
No need to change the first line, keep it `mkspecs/common/clang.conf`. Error: `mkspecs/unsupported/macx-clang-libc++/clang.conf: No such file or directory`